server/tornado_app.py

```diff
@@ -0,0 +1,89 @@
+import functools
+
+from tornado import websocket, web, ioloop
+from tornado.gen import coroutine
+
+import rethinkdb as r
+
+from init_db import get_bigchain
+
+clients = []
+bigchain = get_bigchain()
+
+# from http://blog.hiphipjorge.com/django-and-realtime-using-django-with-tornado-and-rethinkdb/
+r.set_loop_type('tornado')
+
+
+@coroutine
+def print_changes(db_table):
+    conn = yield bigchain.conn
+    feed = yield r.table(db_table).changes().run(conn)
+    while (yield feed.fetch_next()):
+        change = yield feed.next()
+        block = get_block_from_change(change, db_table)
+        for client in clients:
+            for tx in block:
+                # TODO: use REQL for filtering
+                if tx_contains_vk(tx['transaction'], client.username):
+                    client.write_message(change)
+                    break
+
+
+def get_block_from_change(change, db_table):
+    block = []
+    if db_table == 'backlog':
+        if change['old_val']:
+            block.append(change['old_val'])
+        elif change['new_val']:
+            block.append(change['new_val'])
+    elif db_table == 'bigchain':
+        if change['old_val']:
+            block = change['old_val']['block']['transactions']
+        elif change['new_val']:
+            block = change['new_val']['block']['transactions']
+    return block
```

Comment:
Not sure how well this translates into python, but I think this function would be more readable if it were something along the lines of:

```js
get_block_from_change(change, db_table) {
    let blockData;

    if (db_table === 'backlog' || db_table === 'bigchain') {
        blockData = change['old_val'] || change['new_val'];

        if (blockData && db_table === 'bigchain') {
            blockData = blockData['block']['transactions'];
        }
    }

    return blockData ? [blockData] : [];
}